test/ignores.json

```
@@ -27,5 +27,8 @@
   "render-tests/icon-text-fit/text-variable-anchor-tile-map-mode": "skip - mapbox-gl-js does not support tile-mode",
   "render-tests/text-variable-anchor/all-anchors-labels-priority-tile-map-mode": "skip - mapbox-gl-js does not support tile-mode",
   "render-tests/fill-extrusion-pattern/1.5x-on-1x-add-image": "skip - non-deterministic on AMD graphics cards",
-  "render-tests/text-variable-anchor/avoid-edges-tile-map-mode": "skip - mapbox-gl-js does not support tile-mode"
-}
+  "render-tests/text-variable-anchor/avoid-edges-tile-map-mode": "skip - mapbox-gl-js does not support tile-mode",
+  "render-tests/extent/1024-symbol": "skip - MapLibre 1.13.1 baseline",
+  "render-tests/icon-text-fit/enlargen-both": "skip - MapLibre 1.13.1 baseline",
+  "render-tests/line-dasharray/case/round": "skip - MapLibre 1.13.1 baseline"
+}
```

eol (many editors can add these automatically)
```suggestion
}

```